example/paper.md

```diff
@@ -4,7 +4,7 @@ title: >-
 authors:
   - name: Albert Krewinkel
     email: albert@zeitkraut.de
-    affiliation: [1, 2, 4]
+    affiliation: "1, 2, 4"
```

Comment:
Why this change?

Reply:
It appears that we're supposed to be supporting both flavors of affiliation strings, so I wanted to make sure that the example paper reflected one of each way